diff -urN atanks-5.4/src/Makefile atanks-5.4-patched/src/Makefile
--- atanks-5.4/src/Makefile	2012-04-09 03:49:27.000000000 +1100
+++ atanks-5.4-patched/src/Makefile	2012-04-21 15:23:02.562904924 +1100
@@ -105,7 +105,7 @@
 	$(CXX) -c $< -o $@ $(FLAGS) $(LFLAGS) $(OFLAGS) $(CXXFLAGS)
 
 $(OUTPUT): $(OBJECTS) $(MODULES)
-	$(CXX) $(MODULES) -o $(OUTPUT) $(FLAGS) $(LFLAGS) $(LDFLAGS) $(SFLAGS) $(CXXFLAGS)
+	$(CXX) $(FLAGS) $(SFLAGS) $(CXXFLAGS) $(LDFLAGS) $(LFLAGS) $(MODULES) -o $(OUTPUT)
 	# strip $(OUTPUT)
 
 # dependencies:
